fsw/src/crypto.c

```diff
@@ -1753,7 +1800,7 @@ static int32 Crypto_Key_verify(char* ingest,TC_t* tc_frame)
         iv_loc = count;
         for (int y = 0; y < IV_SIZE; y++)
         {   
-            ingest[count++] = tc_frame->tc_sec_header.iv[y];
+            ingest[count++] = *(tc_frame->tc_sec_header.iv)+y;
```

Comment:
Believe this +y should be on the inside of the parens.  Cody and I are reviewing and making changes as we go.

Reply:
agreed.